src/rules/quotePropsRule.ts

```diff
@@ -0,0 +1,103 @@
+import * as Lint from "../lint";
+import * as ts from "typescript";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    /* tslint:disable:object-literal-sort-keys */
+    public static metadata: Lint.IRuleMetadata = {
+        ruleName: "quote-props",
+        description: "Quoting Style for Property Names",
```

Comment:
"Enforces consistent object literal property quote style."

I think the above would be more consistent with [the current descriptions](http://palantir.github.io/tslint/rules/).